monitor/io.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+import numpy as np
+from tempfile import mkstemp
+from shutil import move, rmtree
+
+######### ----------------------------------------###########
+
+def proc_subprocess(executing_arguments, dir):
```

Comment:
[`dir`](https://docs.python.org/2/library/functions.html#dir is a built-in.  Rename to something else (perhaps `log_path`?).